<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Задача 1</string>
     </attribute>
     <widget class="QTextEdit" name="te_Result">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_b">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>146</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_c">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_a">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_Calc">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчитать корни</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Нахождение корней квадратного уравнения ax^2+by+c=0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Задача 2</string>
     </attribute>
     <widget class="QTextEdit" name="teResult2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leB">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>угол</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leAlpha">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leA">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_Calc_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчитать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет стороны треугольника по двум сторонам и углом между ними.</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>121</y>
        <width>141</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbDegrees">
         <property name="text">
          <string>Градусы</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbRadians">
         <property name="text">
          <string>Радианы</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Задача 3</string>
     </attribute>
     <widget class="QPlainTextEdit" name="ptEdit1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="ptEdit2">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbEdit1toEdit2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbEdit2toEdit1">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbResetEdit1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>leA</tabstop>
  <tabstop>leB</tabstop>
  <tabstop>leAlpha</tabstop>
  <tabstop>rbDegrees</tabstop>
  <tabstop>rbRadians</tabstop>
  <tabstop>pb_Calc_2</tabstop>
  <tabstop>te_Result</tabstop>
  <tabstop>le_a</tabstop>
  <tabstop>teResult2</tabstop>
  <tabstop>le_c</tabstop>
  <tabstop>pb_Calc</tabstop>
  <tabstop>le_b</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
